src/main/kotlin/eu/europa/ec/eudi/pidissuer/adapter/out/jose/ValidateJwtProof.kt

```diff
@@ -163,9 +171,7 @@ private fun processor(
             jwtClaimsSetVerifier =
                 DefaultJWTClaimsVerifier<SecurityContext?>(
                     credentialIssuerId.externalForm, // aud
-                    JWTClaimsSet.Builder()
-                        .claim("nonce", expected.nonce)
-                        .build(),
+                    JWTClaimsSet.Builder().build(),
                     setOf("iat", "nonce"),
```

Comment:
Let's remove `nonce`. There is no need to hard-code requirements for nonce at this level